string channel_name                         # User defined name
calibration_msgs/ImagePointStamped pixel    # The settled pixel location
sensor_msgs/Image image                     # The image used to calculate the pixel location
